### PR TITLE
Add Bylaws to the governance repo.

### PR DESCRIPTION
**Pull request by erinishimoticha**
The Board of Directors voted today to approve a set of bylaws in its initial
efforts toward ramping up executive function. We'll record these here for
transparency and ease of record-keeping.

### CODE REVIEW

File: Natives_In_Tech_Bylaws.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+The Board of Directors may amend these Bylaws by majority vote at any regular or special meeting. Written notice setting forth the proposed amendment or summary of the changes to be effected thereby shall be given to each director within the time and the manner provided for the giving of notice of meetings of directors.
+
+Article XII Adoption of By Laws
+===============================
+
+We, the undersigned, are all of the initial directors or incorporators of this corporation, and we consent to, and hereby do, adopt the foregoing Bylaws, consisting of the 13 preceding pages, as the Bylaws of this corporation.
+
+ADOPTED AND APPROVED by the Board of Directors.
+
+**\<Office\>|"Director"** \<Name\> \<email\>, signed this day the XX day of Month, 2020.
```
Comment (arecvlohe) [inline Natives_In_Tech_Bylaws.md]: I will pull this down and add my name and email

**Comment by arecvlohe** [comment]
Git rebase is hard :(

**Comment by erinishimoticha** [comment]
@arecvlohe Looks great to me. Just waiting for @nicksahler!

**Comment by erinishimoticha** [comment]
TODO after merging: Mark the Google Drive bylaws document as deprecated and link here.

File: Natives_In_Tech_Bylaws.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+The Board of Directors may amend these Bylaws by majority vote at any regular or special meeting. Written notice setting forth the proposed amendment or summary of the changes to be effected thereby shall be given to each director within the time and the manner provided for the giving of notice of meetings of directors.
+
+Article XII Adoption of By Laws
+===============================
+
+We, the undersigned, are all of the initial directors or incorporators of this corporation, and we consent to, and hereby do, adopt the foregoing Bylaws, consisting of the 13 preceding pages, as the Bylaws of this corporation.
+
+ADOPTED AND APPROVED by the Board of Directors.
+
+**President** Adam Recvlohe adam@nativesintech.org, signed this day the XX day of Month, 2020.
```
Comment (erinishimoticha) [inline Natives_In_Tech_Bylaws.md]: @arecvlohe, can you add March 6th as the date you signed and force push again?

**Comment by arecvlohe** [comment]
Done

**Review by arecvlohe — APPROVED** [review]

**Review by microfauna — APPROVED** [review]

**Pull request merged**